website/docs/getting-started/setup/instance-configuration/README.mdx

```diff
@@ -0,0 +1,146 @@
+  <TabItem label="Community Edition" value="communityEdition"> 
+
+  You may choose to configure environment variables: through commands or by using a `values.yaml` file.
+
+  * When using command, specify each parameter using `--set key=value[,key=value]` argument to helm upgrade.
+
+```bash
+  helm upgrade --set applicationConfig.<ENV_KEY>=<ENV_VALUE> [ RELEASE ] [ CHART ]  
+```
+  For example, the below command sets the `APPSMITH_SIGNUP_DISABLED` variable to `true` and configures it to disable signup button on Appsmith login screen. 
+
+  * Alternatively, use a `values.yaml` file. You can specify the parameter values in the `applicationConfig` section. 
+
+```yaml
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET: ""
+  APPSMITH_FORM_LOGIN_DISABLED: ""
+  APPSMITH_SIGNUP_DISABLED: "true"
+```
+Once you have made changes to the values file, run the below command.
+
+```bash
+  helm upgrade --values values.yaml appsmith appsmith/appsmith
+```
+
+  </TabItem>
+  <TabItem label="Business Edition" value="businessEdition"> 
+
+   You may choose to configure environment variables: through commands or by using a `values.yaml` file.
+
+  * When using command, specify each parameter using `--set key=value[,key=value]` argument to helm upgrade.
+
+```bash
+  helm upgrade --set applicationConfig.<ENV_KEY>=<ENV_VALUE> [ RELEASE ] [ CHART ]  
+```
+  For example, the below command sets the `APPSMITH_SIGNUP_DISABLED` variable to `true` and configures it to disable signup button on Appsmith login screen. 
+
+  * Alternatively, use a `values.yaml` file. You can specify the parameter values in the `applicationConfig` section. 
+
+```yaml
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET: ""
+  APPSMITH_FORM_LOGIN_DISABLED: ""
+  APPSMITH_SIGNUP_DISABLED: "true"
+```
+Once you have made changes to the values file, run the below command.
+
+```bash
+  helm upgrade --values values.yaml appsmith appsmith-ee/appsmith
+```
+  </TabItem>
+</Tabs> 
+
+## Configure ECS installations
+
+To configure an ECS installation, follow these steps:
+
+1. Navigate to the **ECS console** and select **Task Definitions** on the sidebar.
+2. Click on the Task Definition used by your ECS instance, and hit **Create new revision**.
+3. On the `Task Definition config` page, click the Appsmith **container definition** to edit it. In the **Environment Section**, enter the environment configuration as **key-value pairs** as shown below.
+
+![](/img/ecs-task-env_(1).png)
+
+It's recommended to use **AWS Secrets** for sensitive information, please follow the steps to [create a secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage\_create-basic-secret.html). Use the **ARN** of the secret as the Environment value and set the option to **ValueFrom**.
```

Comment:
I don't think we need to escape this anymore. We're not on Gitbook anymore, and Docusaurus' Markdown parser is quite smart.

```suggestion
It's recommended to use **AWS Secrets** for sensitive information, please follow the steps to [create a secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_create-basic-secret.html). Use the **ARN** of the secret as the Environment value and set the option to **ValueFrom**.
```